docs/_docs/02-ug-configuration.md

```diff
@@ -17,6 +17,6 @@ classes: wide
 | spark.hyperspace.index.lineage.enabled   | false                                                                                    | Add lineage column to index upon creation to track source data file for each index record. Lineage is required to handle deleted files in Hybrid Scan, or to refresh an index in the incremental mode. Adding lineage will increase the size of the index, proportional to the number of distinct source data files the index is built on. | 0.3.0         |
 | spark.hyperspace.index.optimize.fileSizeThreshold   | 256MB                                                                                    | Threshold of size of index files in bytes to optimize. Files with size below this threshold are eligible for merge during index optimization.            | 0.3.0         |
 | spark.hyperspace.index.hybridscan.enabled   | false                                                                                  | Enable Hybrid Scan at query execution time. If enabled, Hyperspace considers an index with appended and/or deleted source data files as a candidate during query optimization, with some additional optimization overhead. | 0.3.0         |
-| spark.hyperspace.index.hybridscan.delete.enabled   | false                                                                                    | (temporary) Enable Hybrid Scan for deleted files. Lineage is required to handle deleted files in Hybrid Scan.  | 0.3.0         |
-| spark.hyperspace.index.hybridscan.delete.maxNumDeleted   | 30                                                                                    | Threshold of the maximum number of deleted files for Hybrid Scan. | 0.3.0         |
+| spark.hyperspace.index.hybridscan.maxAppendedRatio   | 0.3                                                                          |  Ratio threshold for the max. amount of appended data for Hybrid Scan. If there's more appended data than this threshold, Hybrid Scan won't be applied. It's 30% (0.3) by default.   | 0.4.0         |
+| spark.hyperspace.index.hybridscan.maxDeletedRatio   | 0.2                                                                            | Ratio threshold for the max. amount of deleted data for Hybrid Scan. If there's more deleted data than this threshold, Hybrid Scan won't be applied. It's 20% (0.2) by default. Lineage is required to handle deleted files with Hybrid Scan.   | 0.4.0         |
```

Comment:
could you please clarify "amount of appended data"? is it number of files? Is it file size?